.gitignore

```diff
@@ -1 +1,3 @@
+.nyc_output/
+coverage/
```

Comment:
I felt that this was not needed for this project.

Reply:
good call, those directories must have been from an old run on my laptop. I removed it